### PR TITLE
Feat/org

### PR DESCRIPTION
**Pull request by mozwell**
- Add create org page

### CODE REVIEW

File: components/form-item.tsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+  const cls = clsx({
+    'form-control flex': true,
+    'flex-row': props.direction === 'horizontal',
+    [`gap-${props.gap}`]: props.gap,
+  })
+
   return (
-    <div className="form-control w-full max-w-xs">
-      <label className="label">{props.label}</label>
+    <div className={cls + ' ' + restCls}>
```
Comment (renzholy) [inline components/form-item.tsx:19]: `clsx(cls, restCls)`

File: components/form-item.tsx [hunk 2]
```diff
@@ -1,12 +1,23 @@
 import { ReactNode } from 'react'
+import clsx from 'clsx'
 
 export default function FormItem(props: {
   label: string
   children: ReactNode
+  direction?: 'horizontal' | 'vertical'
+  gap?: number
+  classNames?: string
```
Comment (renzholy) [inline components/form-item.tsx:9]: just use `className` here and do not rename it to `restCls`

File: components/nav-bar.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Navbar, Dropdown, Button } from 'react-daisyui'
+import { Navbar, Dropdown, Button, Link } from 'react-daisyui'
```
Comment (renzholy) [inline components/nav-bar.tsx:1]: `import Link from 'next/link'`

File: components/nav-bar.tsx [hunk 4]
```diff
@@ -14,16 +14,24 @@ const ConnectButtonCustom = dynamic(
...
+            <Button className="text-xl normal-case" color="ghost">
+              VotyXYZ
+            </Button>
+          </Link>
         </div>
         <div className="flex-none gap-3">
           <ThemeSwitcher />
-          <Button color="primary">Create an Organization</Button>
+          <Link
+            href="/create-organization"
```
Comment (renzholy) [inline components/nav-bar.tsx:29]: `href="/create"` is enough, no suffix

File: pages/_app.tsx [hunk 5]
```diff
@@ -47,15 +48,25 @@ export default function App({ Component, pageProps }: AppProps) {
     setTheme(theme)
   }, [isDarkMode, setTheme, persistentTheme])
 
+  // To resolve responsiveness in mobile
+  // Why we should put <Head> in this component: https://nextjs.org/docs/messages/no-document-viewport-meta
```
Comment (renzholy) [inline pages/_app.tsx:52]: remove this comment because it's common knowledge

File: pages/create-organization.tsx [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+function CreateSuccess() {
+  return (
+    <div className="flex justify-center flex-col items-center mt-32">
+      <div className="w-20">
+        <GreenCheck />
+      </div>
+      <h1 className="text-3xl md:text-4xl font-bold mb-3 mt-12 text-center">
+        UnknownDAO is created successfully
+      </h1>
+      <Link href="/organization" target="_self" className="hover:no-underline">
```
Comment (renzholy) [inline pages/create-organization.tsx:178]: wrong href

File: pages/create-organization.tsx [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+        <Button color="primary" className="w-fit px-8 mt-16">
+          Enter My Organization
+        </Button>
+      </Link>
+    </div>
+  )
+}
+
+function GreenCheck() {
```
Comment (renzholy) [inline pages/create-organization.tsx:187]: move svg files to another folder

File: pages/create-organization.tsx [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+        </Link>
+      </div>
+    </div>
+  )
+}
+
+function BasicInfo(props: { onPrev: () => void; onNext: () => void }) {
+  const { onPrev, onNext } = props
+  const [organizationName, setOrganizationName] = useState<string>('')
+  const [desc, setDesc] = useState<string>('')
```
Comment (renzholy) [inline pages/create-organization.tsx:118]:
```
const [desc, setDesc] = useState('')
```
`<string>` is not necessary

File: pages/create.tsx [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+import { Button, Steps, Select, Link, Input } from 'react-daisyui'
+import FormItem from '../components/form-item'
+import useSWR from 'swr'
+import { useAccount } from 'wagmi'
+import { createInstance } from 'dotbit'
+import { useRouter } from 'next/router'
+
+function useSteps() {
+  const router = useRouter()
+  return [
```
Comment (renzholy) [inline pages/create.tsx:11]: useMemo

File: pages/create.tsx [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+        Basic Information
+      </Steps.Step>
+      <Steps.Step color={steps > 2 ? 'primary' : undefined}>Done</Steps.Step>
+    </Steps>
+  )
+}
+
+function StepsPage(props: any) {
+  const [stepsQuery, setStepsQuery] = useSteps()
+  const [steps, setSteps] = useState(1)
```
Comment (renzholy) [inline pages/create.tsx:57]: remove `useState`, use `useSteps` only

**Review by renzholy — APPROVED** [review]

File: components/form-item.tsx (hunk 2 above)
Comment (mozwell) [reply components/form-item.tsx]: Fixed.

File: components/nav-bar.tsx (hunk 3 above)
Comment (mozwell) [reply components/nav-bar.tsx]: Fixed.

File: components/nav-bar.tsx (hunk 4 above)
Comment (mozwell) [reply components/nav-bar.tsx]: Fixed.

File: pages/create-organization.tsx (hunk 6 above)
Comment (mozwell) [reply pages/create-organization.tsx]: Fixed.

File: pages/create-organization.tsx (hunk 8 above)
Comment (mozwell) [reply pages/create-organization.tsx]: Fixed.

File: pages/create-organization.tsx (hunk 7 above)
Comment (mozwell) [reply pages/create-organization.tsx]: Will do it later.

**Pull request merged**